chaos/cases/task.go

```diff
@@ -363,6 +365,10 @@ func (t *task) genIncrData(ctx context.Context) (err error) {
 				})
 			}
 			if err = eg.Wait(); err != nil {
+				if utils.IsMySQLError(err, mysql.ErrDupFieldName) {
```

Comment:
`eg.Wait()` will only return the **first** error, but there may be other errors at the same time. How about doing this filter in `eg.Go(func() error {`?